From 7f0ff0ce6b946befdf90ec8939195ec2248352f2 Mon Sep 17 00:00:00 2001
From: Sylvain Munaut <tnt@246tNt.com>
Date: Sun, 13 May 2007 14:16:16 +0200
Subject: [PATCH 12/21] powerpc: BestComm ATA task, move the EXPORT_SYMBOL and use the _GPL version

Also fix a typo bcom_ata_reset vs bcom_ata_reset_bd

Signed-off-by: Sylvain Munaut <tnt@246tNt.com>
---
 arch/powerpc/sysdev/bestcomm/ata.c |   11 +++++------
 1 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/arch/powerpc/sysdev/bestcomm/ata.c b/arch/powerpc/sysdev/bestcomm/ata.c
index c14f727..edc2049 100644
--- a/arch/powerpc/sysdev/bestcomm/ata.c
+++ b/arch/powerpc/sysdev/bestcomm/ata.c
@@ -95,6 +95,7 @@ bcom_ata_init(int queue_len, int maxbufsize)
 
 	return tsk;
 }
+EXPORT_SYMBOL_GPL(bcom_ata_init);
 
 void bcom_ata_rx_prepare(struct bcom_task *tsk)
 {
@@ -108,6 +109,7 @@ void bcom_ata_rx_prepare(struct bcom_task *tsk)
 
 	bcom_set_initiator(tsk->tasknum, BCOM_INITIATOR_ATA_RX);
 }
+EXPORT_SYMBOL_GPL(bcom_ata_rx_prepare);
 
 void bcom_ata_tx_prepare(struct bcom_task *tsk)
 {
@@ -121,6 +123,7 @@ void bcom_ata_tx_prepare(struct bcom_task *tsk)
 
 	bcom_set_initiator(tsk->tasknum, BCOM_INITIATOR_ATA_TX);
 }
+EXPORT_SYMBOL_GPL(bcom_ata_tx_prepare);
 
 void bcom_ata_reset_bd(struct bcom_task *tsk)
 {
@@ -135,20 +138,16 @@ void bcom_ata_reset_bd(struct bcom_task *tsk)
 	var = (struct bcom_ata_var *) bcom_task_var(tsk->tasknum);
 	var->bd_start = var->bd_base;
 }
+EXPORT_SYMBOL_GPL(bcom_ata_reset_bd);
 
 void bcom_ata_release(struct bcom_task *tsk)
 {
 	/* Nothing special for the ATA tasks */
 	bcom_task_release(tsk);
 }
+EXPORT_SYMBOL_GPL(bcom_ata_release);
 
 
-EXPORT_SYMBOL(bcom_ata_init);
-EXPORT_SYMBOL(bcom_ata_rx_prepare);
-EXPORT_SYMBOL(bcom_ata_tx_prepare);
-EXPORT_SYMBOL(bcom_ata_reset);
-EXPORT_SYMBOL(bcom_ata_release);
-
 MODULE_DESCRIPTION("BestComm ATA task driver");
 MODULE_AUTHOR("John Rigby");
 MODULE_LICENSE("GPL v2");
-- 
1.5.1.2

